src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CurrentUserOnly.netcoreapp.Windows.cs

```diff
@@ -120,7 +120,7 @@ public void RunImpersonated(Action action)
     }
 
     /// <summary>
-    /// Tests for the constructors for NamedPipeClientStream
+    /// Negative tests for PipeOptions.CurrentUserOnly in Windows.
```

Comment:
Later I will look at adding the test combination, that is expected to succeed (`PipeOptions.None, PipeOptions.None`) that is missing on Windows for different users. The case `PipeOptions.CurrentUserOnly, PipeOptions.CurrentUserOnly` also should be added in this case, in Windows, we expect both sides to not find each other (prevented by ACL).